datatypes/src/raster/base_raster.rs

```diff
@@ -1,18 +1,20 @@
+use crate::error;
 use crate::raster::{
-    Capacity, Dim, GenericRaster, GeoTransform, GridDimension, GridIndex, GridPixelAccess,
-    GridPixelAccessMut, Pixel, Raster,
+    Capacity, GridDimension, GridIndex, GridPixelAccess, GridPixelAccessMut, Pixel,
 };
 use crate::util::Result;
-use crate::{
-    error,
-    primitives::{BoundingBox2D, SpatialBounded, TemporalBounded, TimeInterval},
-};
 use num_traits::AsPrimitive;
 use serde::{Deserialize, Serialize};
 use snafu::ensure;
 use std::convert::AsRef;
 use std::fmt::Debug;
 
+use super::{Dim1D, Dim2D, Dim3D};
+
+pub type Raster1D<T> = BaseRaster<Dim1D, T, Vec<T>>;
+pub type Raster2D<T> = BaseRaster<Dim2D, T, Vec<T>>;
+pub type Raster3D<T> = BaseRaster<Dim3D, T, Vec<T>>;
+
 #[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
 pub struct BaseRaster<D, T, C>
```

Comment:
we discussed calling the BaseRaster/RasterxD Array and the RasterTile Raster

Reply:
removed BaseRaster. Will name GridArray to Array. However, I think RasterTile should still be called RasterTile.